### PR TITLE
scala 2.12/2.13 crossbuild

### PR DESCRIPTION
**Pull request by gadomsky**

### CODE REVIEW

File: engine/flink/management/src/it/resources/docker/Dockerfile [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+COPY log4j-console.properties /
 
 #TODO: figure out users...
 
 USER root
 RUN chown flink /entrypointWithIP.sh
 
 RUN chown flink /conf.yml
+RUN chown flink /log4j-console.properties
 RUN chmod +x /entrypointWithIP.sh
```
Comment (mproch) [inline engine/flink/management/src/it/resources/docker/Dockerfile:14]: `RUN rm lib/flink-scala_2.12-1.16.0.jar`

I wonder if adding this + downloading [scala 2.13 library](https://search.maven.org/remotecontent?filepath=org/scala-lang/scala-library/2.13.10/scala-library-2.13.10.jar) would be sufficient for scala 2.13 to work?

Comment (gadomsky) [reply engine/flink/management/src/it/resources/docker/Dockerfile:14]: it worked

File: ciRunSbt.sh [hunk 2]
```diff
@@ -10,5 +10,5 @@ else
 fi
 # Tuning of sbt test to prevent travis OOM from killing java. Be aware that using JAVA_OPTS won't work on travis because
 # it has own alias for sbt: https://www.scala-sbt.org/1.x/docs/Travis-CI-with-sbt.html#Custom+JVM+options
-echo "Executing: sbt -J-Xss6M -J-Xms1500M -J-Xmx1500M -J-XX:ReservedCodeCacheSize=256M -J-XX:MaxMetaspaceSize=2500M $ARGS"
-sbt -J-Xss6M -J-Xms1500M -J-Xmx1500M -J-XX:ReservedCodeCacheSize=256M -J-XX:MaxMetaspaceSize=2500M $ARGS
+echo "Executing: sbt -J-Xss6M -J-Xms2g -J-Xmx2g -J-XX:ReservedCodeCacheSize=256M -J-XX:MaxMetaspaceSize=5g $ARGS"
```
Comment (mproch) [inline ciRunSbt.sh:13]: well... this deserves at least a TODO IMHO ;)

**Review by mproch — COMMENTED** [review]
Cool, added some comments

File: engine/flink/management/src/it/scala/pl/touk/nussknacker/engine/management/DockerTest.scala [hunk 3]
```diff
@@ -47,12 +47,27 @@ trait DockerTest extends BeforeAndAfterAll with ForAllTestContainer with Extreme
...
       val resource = IOUtils.toString(rezz)
-      val withVersionReplaced = resource.replace("${scala.major.version}", ScalaMajorVersionConfig.scalaMajorVersion)
-      image.withFileFromString(file, withVersionReplaced)
+
+      val flinkLibTweakCommand = ScalaMajorVersionConfig.scalaMajorVersion match {
+        case "2.12" => ""
+        case "2.13" =>
+          val scalaV = util.Properties.versionNumberString
+          s"""
+            |RUN rm $$FLINK_HOME/lib/flink-scala_2.12-1.16.0.jar
```
Comment (mproch) [inline engine/flink/management/src/it/scala/pl/touk/nussknacker/engine/management/DockerTest.scala:60]: maybe `rm flink-scala*.jar` so we don't care about Flink upgrades?

**Review by mproch — APPROVED** [review]
Looks pretty cool, thx :+1:

**Pull request merged**